.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,7 +5,7 @@
 Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:
 
  - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
- - [ ] Mention the appropriate issue in your description (for example: "addresses #123), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment "fixes #<ISSUE NUMBER>" instead.
+ - [ ] Mention the appropriate issue in your description (for example: "`addresses #123`"), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment "`fixes #<ISSUE NUMBER>`" instead.
```

Comment:
```suggestion
 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
```

Nit: We probably don't need the quotes anymore